### ISSUE TITLE
openvino model

### ISSUE THREAD

**Issue by kleMor** [snakers4/silero-models#199]
Did you consider also to provide an openvino model?

Best regards,
M

**Comment by snakers4** [comment]
Yeah, we considered TF hub and open vino hub, and we found their terms and conditions not very lucrative for us.
